air/src/utils.rs

```diff
@@ -44,6 +51,107 @@ impl<E: FieldElement> EvaluationResult<E> for Vec<E> {
     }
 }
 
+// COMMON STACK CONSTRAINTS
+// ================================================================================================
```

Comment:
These functions should probably go under stack module (vs. general utils module). Also, I'm not sure this is the right way to do it. I'd rather remove these from this PR, and tackle general constraints in the next PR.

Reply:
Removed them for the time being in the latest PR.